src/main/java/mate/academy/service/MovieServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.service;
+
+import mate.academy.dao.MovieDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Injector;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+    @Inject
+    private static final MovieDao movieDao = (MovieDao) injector.getInstance(MovieDao.class);
```

Comment:
```suggestion
    @Inject
    private MovieDao movieDao;
```
create a MovieService with injector in the main class

Reply:
Thanks